src/main/java/org/gridsuite/loadflow/server/LoadFlowService.java

```diff
@@ -86,15 +86,14 @@ LoadFlowResult loadFlow(UUID networkUuid, String variantId, List<UUID> otherNetw
         LoadFlowResult result;
         String providerToUse = provider != null ? provider : defaultProvider;
 
-        Reporter reporter;
+        Reporter rootReporter = Reporter.NO_OP;
+        Reporter reporter = Reporter.NO_OP;
         if (reportInfos.getReportId() != null) {
-            String name =
-                    (reportInfos.getReportName() == null ? "loadflow" : reportInfos.getReportName())
-                    + " (" + providerToUse + ")";
-            reporter = new ReporterModel(name, name);
-        } else {
-            reporter = Reporter.NO_OP;
+            String rootReporterId = reportInfos.getReportName() +  "@" + LOAD_FLOW_TYPE_REPORT;
```

Comment:
Maybe add a check if reportInfos.getReportName() is null (in theory, it can be the case because reportName is a **not required** parameter ...)

Reply:
yes